schema/soo/ofc/custom_properties/common.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 FoxyUtils ehf. All rights reserved.
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
```

Comment:
Few more files going back in time.  Just check.

Reply:
I suppose it is ok, it will be changed soon anyway.